vstools/functions/normalize.py

```diff
@@ -237,7 +237,7 @@ def normalize_ranges(clip: vs.VideoNode, ranges: FrameRangeN | FrameRangesN) ->
             if start is None:
                 start = 0
             if end is None:
-                end = clip.num_frames - 1
```

Comment:
this is not a bug, please revert

Reply:
The examples in this function are incorrect then. I can revert this change and update the examples instead. (The tests are identical to the examples.)